### PR TITLE
Alerts 2019 1

### PR DESCRIPTION
**Pull request by imnotashrimp**
# What changed

Revise the docs for the new alerts UI

----

## Pages to review

- [x] https://deploy-preview-338--logz-docs.netlify.com/user-guide/alerts/configure-an-alert.html

## Remaining work

- [x] Technical review
- [x] Copy Review
- [x] Redirects: https://deploy-preview-338--logz-docs.netlify.com/user-guide/alerts/configure-alerts.html => configure-an-alert.html

## Post launch

**Do not remove** - To be completed by the docs team upon merge:

no tasks

### CODE REVIEW

File: _source/user-guide/alerts/configure-alerts.md [hunk 1]
```diff
@@ -10,101 +10,166 @@ contributors:
   - imnotashrimp
 ---
 
-To create a new alert, you'll start in Kibana so you can test the query you want to use. After that, you'll continue to the Create a New Alert page, where you can configure the alert and notification settings.
+When you set out to create an alert,
+you'll need to start with a query.
+If you haven't formed your query yet,
+we recommend starting off in Kibana
+so you can see the results of the query in real time.
```
Comment (AmirTugi) [inline _source/user-guide/alerts/configure-alerts.md]: Just making sure you have it in a note to change this once we have live preview in the alerts page :)

File: _source/user-guide/alerts/configure-alerts.md [hunk 2]
```diff
@@ -10,101 +10,166 @@ contributors:
...
+
+Otherwise, if you already have a query and filters,
+you can create a blank alert
+from the top menu (**Alerts & Events > New alert**)
+or from the _Alert definitions_ page
```
Comment (AmirTugi) [inline _source/user-guide/alerts/configure-alerts.md]: It's also available from the `triggered alerts` page

File: _source/user-guide/alerts/configure-alerts.md [hunk 3]
```diff
@@ -10,101 +10,166 @@ contributors:
...
 
-If you use an invalid query, the alert will be automatically disabled. Run your query in Kibana so you can be sure you're getting the expected results.
+Your query is an important source of information
+when you're investigating a triggered alert.
+Shorten your query by using filters whenever you can.
+
+If you use an invalid query,
+the alert will be automatically disabled.
+To make sure it's valid,
+test your query in Kibana.
```
Comment (AmirTugi) [inline _source/user-guide/alerts/configure-alerts.md]: We have the ability in the alerts page to say that the query is invalid, and the user won't be able to save the alert.
@imnotashrimp I don't think we've gone over the error message if the invalid syntax.

File: _source/user-guide/alerts/configure-alerts.md [hunk 4]
```diff
@@ -10,101 +10,166 @@ contributors:
...
 
-In the **Select Field** list,
-choose a field to group by.
+With _Group by_, you can choose up to 3 fields to group.
+The alert will return the aggregated results for each group.
 
-To limit the available fields,
-choose a log type from the **Filter by type** list.
-To show fields for all log types, choose **Clear filter**.
+Careful, though: The order of these fields matters.
```
Comment (AmirTugi) [inline _source/user-guide/alerts/configure-alerts.md]: Classic use-case of a warning box :D

File: _source/user-guide/alerts/configure-alerts.md [hunk 5]
```diff
@@ -10,101 +10,166 @@ contributors:
...
+If you want to send notifications or emails when the alert is triggered,
+choose notification endpoints.
+This isn't required, though—triggered alerts are still logged and searchable in Kibana.
+
+Choose the endpoints or email addresses to notify under _Who to send it to_.
 If you need help adding a new endpoint,
-see [Alert endpoints]({{site.baseurl}}/user-guide/integrations/endpoints.html).
+see [_Notification endpoints_]({{site.baseurl}}/user-guide/integrations/endpoints.html).
 
 Choose a time period to suppress notifications.
```
Comment (AmirTugi) [inline _source/user-guide/alerts/configure-alerts.md]: This feels like you changed context too quickly.
I guess a small intro will be nice here "If you think you will be getting this alert's notification too often, ..."

File: _source/user-guide/alerts/configure-alerts.md [hunk 6]
```diff
@@ -10,101 +10,166 @@ contributors:
...
-##### Choose an output format
+###### Output format
 
-Choose an **Output**.
+![Output table]({{site.baseurl}}/images/alerts/output-table.png)
 
-<video autoplay loop>
-    <source src="{{site.baseurl}}/videos/alerts/alerts--custom-format.mp4" type="video/mp4" />
-</video>
+You can choose to send **All fields** in JSON format,
```
Comment (AmirTugi) [inline _source/user-guide/alerts/configure-alerts.md]: Those will soon change.
We need to be synced with each other to validate the docs.

File: _source/user-guide/alerts/configure-alerts.md [hunk 7]
```diff
@@ -10,101 +10,166 @@ contributors:
...
+Choose the endpoints or email addresses to notify under _Who to send it to_.
 If you need help adding a new endpoint,
-see [Alert endpoints]({{site.baseurl}}/user-guide/integrations/endpoints.html).
+see [_Notification endpoints_]({{site.baseurl}}/user-guide/integrations/endpoints.html).
 
 Choose a time period to suppress notifications.
 
-When notifications are suppressed, Logz.io will continue to log triggered alerts without sending notifications. You can search triggered alert logs at any time.
+When notifications are suppressed,
+Logz.io will continue to log triggered alerts without sending notifications.
```
Comment (idohalevi) [inline _source/user-guide/alerts/configure-alerts.md]: Why is it in a note box? no need in my opinion

File: _source/user-guide/alerts/configure-alerts.md [hunk 8]
```diff
@@ -10,101 +10,166 @@ contributors:
...
+You can search triggered alert logs at any time.
 {:.info-box.note}
 
-##### Choose an output format
+###### Output format
```
Comment (idohalevi) [inline _source/user-guide/alerts/configure-alerts.md]: Only relevant when they added a notification endpoint

File: _source/user-guide/alerts/configure-alerts.md [hunk 9]
```diff
@@ -10,101 +10,166 @@ contributors:
...
-#### To start a new alert
+#### To create an alert in Kibana
 
-![Kibana query bar]({{site.baseurl}}/images/kibana/kibana--query-bar.png)
+![Kibana 7 search bar]({{site.baseurl}}/images/kibana/kibana-7-search-bar.png)
 
-##### Set your query in Kibana
+Refine your search query and filters,
+and review the returned logs
+to make sure you got the expected results.
```
Comment (idohalevi) [inline _source/user-guide/alerts/configure-alerts.md]: We have a char limit for the search box

File: _source/user-guide/alerts/configure-alerts.md [hunk 10]
```diff
@@ -10,101 +10,166 @@ contributors:
...
+![Alert description and tags]({{site.baseurl}}/images/alerts/description-and-tags.png)
 
-If you want to receive notifications or emails when the alert is triggered,
-choose an alert endpoint.
-If you don't choose an endpoint,
-triggered alerts will still be logged:
+The **Description** is visible on the _Alert definitions_ page.
+It's also included with emails and Slack messages when the alert is triggered.
+We recommend making your description helpful to recipients,
+like telling them how to fix the issues that led to the alert.
```
Comment (idohalevi) [inline _source/user-guide/alerts/configure-alerts.md]: We have a char limit for the description box

File: _source/user-guide/alerts/configure-alerts.md [hunk 11]
```diff
@@ -10,101 +10,166 @@ contributors:
...
+If you haven't formed your query yet,
+we recommend starting off in Kibana
+so you can see the results of the query in real time.
+
+Otherwise, if you already have a query and filters,
+you can create a blank alert
+from the top menu (**Alerts & Events > New alert**)
+or from the _Alert definitions_ page
 
 Community plans have a maximum of 50 alerts.
```
[the remark this answers is not in the document]
Comment (imnotashrimp) [reply _source/user-guide/alerts/configure-alerts.md]:
```suggestion
Community plans can have up to 50 alerts enabled at a time.
```

**Comment by afishler** [comment]
The custom JSON option is not supported yet

image (24)

**Comment by afishler** [comment]
The sorting explanation is not so clear and also the regex usage is very complicated and probably needs to be explained in detail

image (25)

File: _source/user-guide/alerts/configure-alerts.md (hunk 4 above)
Comment (AmiStrn) [reply _source/user-guide/alerts/configure-alerts.md]: Also - group by order has NO effect on results. so this whole section is puzzling to me.

Comment (imnotashrimp) [reply _source/user-guide/alerts/configure-alerts.md]: @AmiStrn That's contrary to everything I've been told so far. Can you check and confirm that please?

File: _source/user-guide/alerts/configure-alerts.md (hunk 7 above)
Comment (imnotashrimp) [reply _source/user-guide/alerts/configure-alerts.md]: This was from user feedback a while back. Apparently folks were confused about what suppressing notifications actually meant. I'll keep this as is unless there's a good reason to change.

File: _source/user-guide/alerts/configure-alerts.md (hunk 1 above)
Comment (imnotashrimp) [reply _source/user-guide/alerts/configure-alerts.md]: Duly noted :)

File: _source/user-guide/alerts/configure-alerts.md (hunk 9 above)
Comment (imnotashrimp) [reply _source/user-guide/alerts/configure-alerts.md]: I'll keep an ear out if users are hitting this limit.

File: _source/user-guide/alerts/configure-alerts.md (hunk 10 above)
Comment (imnotashrimp) [reply _source/user-guide/alerts/configure-alerts.md]: Even less likely users will hit it. Leaving this as is.

File: _source/user-guide/alerts/configure-alerts.md (hunk 2 above)
Comment (imnotashrimp) [reply _source/user-guide/alerts/configure-alerts.md]: Creating an alert? Not that I see. Only for editing.

File: _source/user-guide/alerts/configure-alerts.md (hunk 3 above)
Comment (imnotashrimp) [reply _source/user-guide/alerts/configure-alerts.md]: Wrote a string for the error message just now, sent privately. 

@AmirTugi So I can remove this bit completely then? Users are prevented from saving an invalid query?

File: _source/user-guide/alerts/configure-alerts.md (hunk 5 above)
Comment (imnotashrimp) [reply _source/user-guide/alerts/configure-alerts.md]: Done.

File: _source/user-guide/alerts/configure-alerts.md (hunk 6 above)
Comment (imnotashrimp) [reply _source/user-guide/alerts/configure-alerts.md]: Noted.

File: _source/user-guide/alerts/configure-alerts.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (imnotashrimp) [reply _source/user-guide/alerts/configure-alerts.md]: Done. In next push

File: _source/user-guide/alerts/configure-alerts.md [hunk 12]
```diff
@@ -10,101 +10,166 @@ contributors:
...
+from a search written in Kibana Query Language (KQL).
+{:.info-box.important}
 
-Click **Create Alert** (to the right of the query bar).
-The _Create Alert Definition_ page is shown.
+Once you're happy with your alert,
+click **Create alert** (above the query bar).
+You're taken to the _New alert_ page.
 
 Continue with [To configure an alert](#to-configure-an-alert).
```
[the remark this answers is not in the document]
Comment (imnotashrimp) [reply _source/user-guide/alerts/configure-alerts.md]: In the next push

**Comment by imnotashrimp** [comment]
> The custom JSON option is not supported yet
> 
> image (24)

Right you are. Fixed in the next push.

File: _source/user-guide/alerts/configure-alerts.md (hunk 4 above)
Comment (idohalevi) [reply _source/user-guide/alerts/configure-alerts.md]: @AmiStrn In the old page, the group-by had an effect on the results.

Comment (imnotashrimp) [reply _source/user-guide/alerts/configure-alerts.md]: Do we have an answer on this?
@idohalevi @AmiStrn @AmirTugi @schwin007

Comment (imnotashrimp) [reply _source/user-guide/alerts/configure-alerts.md]: > Classic use-case of a warning box :D

Wasn't going to, but now that I can take out the one above it, I will!

Comment (imnotashrimp) [reply _source/user-guide/alerts/configure-alerts.md]: Just received word that order does indeed matter. The text will remain.

File: _source/user-guide/alerts/configure-alerts.md (hunk 3 above)
Comment (imnotashrimp) [reply _source/user-guide/alerts/configure-alerts.md]: Removed

File: _source/user-guide/alerts/configure-alerts.md (hunk 8 above)
Comment (imnotashrimp) [reply _source/user-guide/alerts/configure-alerts.md]: Fixed

**Comment by imnotashrimp** [comment]
@idohalevi are we live?

**Comment by idohalevi** [comment]
> @idohalevi are we live?

нет

**Pull request merged**